### PR TITLE
ESP32-S2-DevKitM

### PR DESCRIPTION
**Pull request by mulcmu**
Added symbol and footprint for the [ESP32-S2-DevKitM](https://docs.espressif.com/projects/esp-idf/en/latest/esp32s2/hw-reference/esp32s2/user-guide-devkitm-1-v1.html) dev board.  

Updated readme.

ESP32-S2-DevKitM
symbol
.

### CODE REVIEW

**Comment by mulcmu** [comment]
I think it is complete.   I double checked it again and made two small tweaks to pad shapes in 8ae8101.

**Comment by pedrominatel** [comment]
Thanks for your contribution @mulcmu!

I'll prepare the review and merge it ASAP!

File: footprints/Espressif.pretty/ESP32-S2-DevKitM.kicad_mod [hunk 1]
```diff
@@ -0,0 +1,422 @@
...
+  )
+  (fp_text user "GPIO40/MTDO" (at 14.162619 20.31728 unlocked) (layer "B.SilkS")
+    (effects (font (size 0.7 0.7) (thickness 0.15)) (justify right mirror))
+    (tstamp 1eeecfbb-3527-478e-b14e-f1a92259fe26)
+  )
+  (fp_text user "GPIO45" (at 17.662619 7.61728 unlocked) (layer "B.SilkS")
+    (effects (font (size 0.7 0.7) (thickness 0.15)) (justify right mirror))
+    (tstamp 2671210c-343e-42c5-bfa0-6397ac0b7e06)
+  )
+  (fp_text user "GPIO19" (at 17.662619 48.25728 unlocked) (layer "B.SilkS")
```
Comment (pedrominatel) [inline footprints/Espressif.pretty/ESP32-S2-DevKitM.kicad_mod:59]: Consider adding the D- to the silkscreen.

File: footprints/Espressif.pretty/ESP32-S2-DevKitM.kicad_mod [hunk 2]
```diff
@@ -0,0 +1,422 @@
...
+  )
+  (fp_text user "GPIO39/MTCK" (at 14.195952 22.85728 unlocked) (layer "B.SilkS")
+    (effects (font (size 0.7 0.7) (thickness 0.15)) (justify right mirror))
+    (tstamp c5a10efe-7a10-4b9e-9cd8-a54114330eab)
+  )
+  (fp_text user "GPIO5" (at 4.816428 15.23728 unlocked) (layer "B.SilkS")
+    (effects (font (size 0.7 0.7) (thickness 0.15)) (justify left mirror))
+    (tstamp d7bbb03c-7336-4248-b4d9-336c31f2f8c4)
+  )
+  (fp_text user "GPIO20" (at 17.662619 45.71728 unlocked) (layer "B.SilkS")
```
Comment (pedrominatel) [inline footprints/Espressif.pretty/ESP32-S2-DevKitM.kicad_mod:151]: Consider adding the D+ to the silkscreen.

**Review by pedrominatel — CHANGES_REQUESTED** [review]
Looks good to me!
Please consider the changes I've suggested and try to keep only GND with the rectangle pad shape.

Thanks!

**Comment by mulcmu** [comment]
All the review comments were addressed in fa376b7

**Review by pedrominatel — APPROVED** [review]
LGTM!

**Pull request merged**